scipy/stats/_multivariate.py

```diff
@@ -5358,8 +5358,8 @@ def _process_parameters(row, col):
         not contain negative or non-integer entries, and that the sums over
         both vectors are equal.
         """
-        r = np.array(row, dtype=np.double, copy=True)
-        c = np.array(col, dtype=np.double, copy=True)
+        r = np.array(row, dtype=np.int64, copy=True)
+        c = np.array(col, dtype=np.int64, copy=True)
```

Comment:
We do this because the internal C code expects int64 values and row-major order (for C). We enforce this here. I suppose that `copy=True` is not strictly necessary, since `row` and `col` are never mutated, but it can't hurt to have that extra safety and it follows the example of other distributions.

Reply:
Sounds good, thanks for the clarification.